src/test/regress/sql/multi_extension.sql

```diff
@@ -44,6 +55,7 @@ ALTER EXTENSION citus UPDATE TO '6.0-10';
 ALTER EXTENSION citus UPDATE TO '6.0-11';
 ALTER EXTENSION citus UPDATE TO '6.0-12';
 ALTER EXTENSION citus UPDATE TO '6.0-13';
+ALTER EXTENSION citus UPDATE TO '6.0-14';
 
```

Comment:
So the interesting part is that the upgrade script I'd originally written failed because it explicitly used `public.<function_name>`. A bare `CREATE EXTENSION` apparently created those "bad" functions in the `public` schema (because of the `RESET search_path` call), but individual `ALTER EXTENSION` calls look as though they run inheriting the `search_path` of the caller maybe?!?

The immediate issue here is that I think we should devise some sort of pattern for writing our upgrade files (do we _always_ wrap them in a `SET search_path TO pg_catalog`/`RESET search_path`?)…

The lingering question of mine that this brings up is I've never really understood why we put anything in `pg_catalog` in the first place. Sure, there are historic reasons for it, but why not be a fully relocatable extension, or create a `citus` schema and put everything there? That's always bothered me, TBH.